cmake/FileList.cmake

```diff
@@ -315,3 +315,7 @@ set(ALLEGRO_INCLUDE_ALLEGRO_ANDROID_FILES
 set(ALLEGRO_INCLUDE_ALLEGRO_PLATFORM_FILES_GENERATED
     include/allegro5/platform/alplatf.h
     )
+    
+set(ALLEGRO_INCLUDE_ALLEGRO_X_FILES
+    include/allegro5/allegro_x.h
+    )
```

Comment:
Only reason I did that was I noticed that true/false can be not declared under older standards, but I would figure the rest of the codebase is using true/false anyway so I'm not sure why I bothered!